rules/windows/powershell/powershell_network_sniffing.yml

```diff
@@ -0,0 +1,27 @@
+title: Network Sniffing
+status: experimental
+description: Network sniffing refers to using the network interface on a system to monitor or capture information sent over a wired or wireless connection. An adversary may place a network interface into promiscuous mode to passively access data in transit over the network, or use span ports to capture a larger amount of data.
+author: Timur Zinniatullin, oscd.community
+references:
+    - https://attack.mitre.org/techniques/T1040/
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1040/T1040.yaml
+logsource:
+    product: windows
+    service: powershell

```

Comment:
@zinint regardless of the way of execution (there are plenty, basically, like powershell, reg, service, task etc) we will always see process execution events with `Image` and `CommandLine` fields. 
This way, since you've already created a [special rule](https://github.com/Neo23x0/sigma/blob/b565398bc5a3b53a0178ffa8be77a5a8cfe363e9/rules/windows/process_creation/win_network_sniffing.yml) for the same stuff but for `process_creation` category, and there is no specific powershell things in the rule, I would consider to delete it %)

Reply:
Of course, my bad, sorry, deleted.